src/app/loadout-builder/mod-utils.ts

```diff
@@ -55,13 +65,33 @@ function stringifyMods(permutation: (PluggableInventoryItemDefinition | null)[])
 }
 
 interface ItemEnergy {
+  /** The energy currently used by slot dependant mods. */
   used: number;
+  /** The original energy capacity of the item. */
   originalCapacity: number;
+  /** The derived energy capacity of the item after considering armour upgrade options. */
```

Comment:
Does this mean it's the maximum energy you could get to given the selected upgrade option?

Reply:
Yes I will change it to that